example/50-seed.yaml

```diff
@@ -0,0 +1,28 @@
+# Seed cluster registration manifest into which the control planes of Shoot clusters will be deployed.
+---
+apiVersion: core.gardener.cloud/v1alpha1
+kind: Seed
+metadata:
+  name: my-seed
+spec:
+  provider:
+    type: <provider-name>  # e.g., aws, azure, gcp, ...
+    region: europe-1
+  secretRef:
+    name: seed-secret
+    namespace: garden
+  dns:
+    ingressDomain: dev.my-seed.example.com
+  networks: # Seed and Shoot networks must be disjunct
+    nodes: 10.240.0.0/16
+    pods: 10.241.128.0/17
+    services: 10.241.0.0/17
+  blockCIDRs:
+  - 169.254.169.254/32
+  protected: false # if true then only shoots in the `garden` namespace can use this seed
```

Comment:
instead of `protected` / `visible`, how about `taints` as we have it in the k8s nodes API?

Reply:
+1, nice idea, will do